packages/vscode-extension/src/officeChat/commands/create/helper.ts

```diff
@@ -95,11 +112,11 @@ export function getOfficeTemplateMetadata(): ProjectMetadata[] {
 export async function showOfficeSampleFileTree(
   projectMetadata: ProjectMetadata,
   response: ChatResponseStream
-): Promise<string> {
+): Promise<object> {
```

Comment:
Define a object struct { path: string, host: string } instead of returning an object

Reply:
will fix, thanks